[bug] Fails To Start In SMP

**describe the bug**
Mod seems to work perfectly in single-player, but when on a dedicated server the server fails to start. Looks like maybe it is trying to call client apis from the server?

**log**
```
[14:43:16] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'bodaciousberries'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.13.2.jar:?]
	at net.minecraft.server.Main.main(Main.java:106) [server-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.2.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.13.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:66) [fabric-loader-0.13.2.jar:?]
	at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:64) [fabric-1.18.1-0.13.2.jar:0.10.2]
Caused by: java.lang.NoClassDefFoundError: net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap
	at io.ix0rai.bodaciousberries.block.BasicBerryBush.<init>(BasicBerryBush.java:83) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.block.ChorusBerryBush.<init>(ChorusBerryBush.java:12) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.registry.Bushes.<clinit>(Bushes.java:28) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.Bodaciousberries.onInitialize(Bodaciousberries.java:20) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.13.2.jar:?]
	... 16 more
Caused by: java.lang.ClassNotFoundException: net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap
	at java.net.URLClassLoader.findClass(URLClassLoader.java:445) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:587) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158) ~[fabric-loader-0.13.2.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at io.ix0rai.bodaciousberries.block.BasicBerryBush.<init>(BasicBerryBush.java:83) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.block.ChorusBerryBush.<init>(ChorusBerryBush.java:12) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.registry.Bushes.<clinit>(Bushes.java:28) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at io.ix0rai.bodaciousberries.Bodaciousberries.onInitialize(Bodaciousberries.java:20) ~[bodaciousberries-1.1.1+mc1.18.1.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.13.2.jar:?]
	... 16 more
```

**version**
`bodaciousberries-1.1.1+mc1.18.1.jar`

--- a/gradle.properties
+++ b/gradle.properties
@@ -8,7 +8,7 @@ yarn_mappings=1.18.1+build.1
 loader_version=0.12.10
 
 # mod Properties
-mod_version = 1.1.1+mc1.18.1
+mod_version = 1.2.0+mc1.18.1
 maven_group = bodaciousberries.bodaciousberries
 archives_base_name = bodaciousberries
 

--- a/gradle.properties
+++ b/gradle.properties
@@ -8,7 +8,7 @@ yarn_mappings=1.18.1+build.1
 loader_version=0.12.10
 
 # mod Properties
-mod_version = 1.2.0+mc1.18.1
+mod_version = 1.1.2+mc1.18.1
 maven_group = bodaciousberries.bodaciousberries
 archives_base_name = bodaciousberries
 

--- a/src/main/java/io/ix0rai/bodaciousberries/block/BasicBerryBush.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/block/BasicBerryBush.java
@@ -3,12 +3,10 @@ package io.ix0rai.bodaciousberries.block;
 import io.ix0rai.bodaciousberries.registry.Bushes;
 import io.ix0rai.bodaciousberries.registry.Sounds;
 import io.ix0rai.bodaciousberries.util.BerryTypeConfigurationException;
-import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.PlantBlock;
 import net.minecraft.block.ShapeContext;
-import net.minecraft.client.render.RenderLayer;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.EntityType;
 import net.minecraft.entity.LivingEntity;
@@ -41,8 +39,9 @@ public class BasicBerryBush extends PlantBlock implements BerryBush {
     protected static final int GROW_CHANCE = 5;
     protected static final int MAX_BERRY_AMOUNT = 3;
 
-    protected Item berryType;
     public static final IntProperty AGE = IntProperty.of("age", 0 ,10);
+
+    protected Item berryType;
     protected final int maxAge;
     protected final VoxelShape smallShape;
     protected final VoxelShape largeShape;
@@ -70,17 +69,12 @@ public class BasicBerryBush extends PlantBlock implements BerryBush {
      * @param sizeChangeAge the age when the bush switches from smallShape to largeShape, this will also be the age it resets to when berries are picked
      */
     public BasicBerryBush(Item berryType, int maxAge, VoxelShape smallShape, VoxelShape largeShape, int sizeChangeAge) {
-        //add nonOpaque to settings to ensure that the bush isn't considered a solid block when rendering
         super(Bushes.BERRY_BUSH_SETTINGS);
         this.berryType = berryType;
         this.maxAge = maxAge;
         this.smallShape = smallShape;
         this.largeShape = largeShape;
         this.sizeChangeAge = sizeChangeAge;
-        //set default age to 0
-        this.setDefaultState((this.stateManager.getDefaultState()).with(AGE, 0));
-        //ensure cutout texture is rendered
-        BlockRenderLayerMap.INSTANCE.putBlock(this, RenderLayer.getCutout());
     }
 
     /**

--- a/src/main/java/io/ix0rai/bodaciousberries/block/BerryVine.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/block/BerryVine.java
@@ -2,11 +2,9 @@ package io.ix0rai.bodaciousberries.block;
 
 import io.ix0rai.bodaciousberries.registry.Bushes;
 import io.ix0rai.bodaciousberries.util.BerryTypeConfigurationException;
-import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.VineBlock;
-import net.minecraft.client.render.RenderLayer;
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
@@ -26,16 +24,15 @@ import java.util.Random;
 @SuppressWarnings("deprecation")
 public class BerryVine extends VineBlock implements BerryBush {
     protected static final int MAX_AGE = 3;
-    public static final IntProperty AGE = IntProperty.of("age", 0, MAX_AGE);
     protected static final int MAX_BERRY_AMOUNT = 3;
+
+    public static final IntProperty AGE = IntProperty.of("age", 0, MAX_AGE);
 
     protected Item berryType;
 
     public BerryVine(Item berry) {
         super(Bushes.BERRY_BUSH_SETTINGS);
         this.berryType = berry;
-        //ensure cutout texture is rendered
-        BlockRenderLayerMap.INSTANCE.putBlock(this, RenderLayer.getCutout());
     }
 
     @Override

--- a/src/main/java/io/ix0rai/bodaciousberries/block/DoubleBerryBush.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/block/DoubleBerryBush.java
@@ -2,11 +2,9 @@ package io.ix0rai.bodaciousberries.block;
 
 import io.ix0rai.bodaciousberries.registry.Bushes;
 import io.ix0rai.bodaciousberries.util.BerryTypeConfigurationException;
-import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.TallPlantBlock;
-import net.minecraft.client.render.RenderLayer;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.LivingEntity;
 import net.minecraft.entity.player.PlayerEntity;
@@ -28,10 +26,11 @@ import java.util.Random;
 
 @SuppressWarnings("deprecation")
 public class DoubleBerryBush extends TallPlantBlock implements BerryBush {
-    protected static final Vec3d DOUBLE_BUSH_SLOWING_VECTOR = new Vec3d(0.7D, 0.9D, 0.7D);
     //berry age is hard capped at 3 for double bushes
-    protected static final int MAX_AGE = 3;
+    public static final int MAX_AGE = 3;
     public static final IntProperty AGE = IntProperty.of("age", 0, MAX_AGE);
+
+    protected static final Vec3d DOUBLE_BUSH_SLOWING_VECTOR = new Vec3d(0.7D, 0.9D, 0.7D);
     protected static final int MAX_BERRY_AMOUNT = 6;
 
     protected Item berryType;
@@ -39,8 +38,6 @@ public class DoubleBerryBush extends TallPlantBlock implements BerryBush {
     public DoubleBerryBush(Item berryType) {
         super(Bushes.BERRY_BUSH_SETTINGS);
         this.berryType = berryType;
-        //ensure cutout texture is rendered
-        BlockRenderLayerMap.INSTANCE.putBlock(this, RenderLayer.getCutout());
     }
 
     @Override

--- a/src/main/java/io/ix0rai/bodaciousberries/client/BodaciousberriesClient.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/client/BodaciousberriesClient.java
@@ -1,0 +1,19 @@
+package io.ix0rai.bodaciousberries.client;
+
+import io.ix0rai.bodaciousberries.registry.Berries;
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
+import net.minecraft.client.render.RenderLayer;
+
+@Environment(EnvType.CLIENT)
+public class BodaciousberriesClient implements ClientModInitializer {
+    @Override
+    public void onInitializeClient() {
+        for (var entry : Berries.BERRY_BUSHES.entrySet()) {
+            //ensure bush is rendered with a cutout
+            BlockRenderLayerMap.INSTANCE.putBlock(entry.getKey().getBaseState().getBlock(), RenderLayer.getCutout());
+        }
+    }
+}

--- a/src/main/java/io/ix0rai/bodaciousberries/registry/Berries.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/registry/Berries.java
@@ -7,6 +7,8 @@ import io.ix0rai.bodaciousberries.block.GrowingBerryBush;
 import io.ix0rai.bodaciousberries.registry.items.ChorusBerries;
 import io.ix0rai.bodaciousberries.registry.items.GojiBerries;
 import io.ix0rai.bodaciousberries.registry.items.Rainberry;
+import net.fabricmc.fabric.api.blockrenderlayer.v1.BlockRenderLayerMap;
+import net.minecraft.client.render.RenderLayer;
 import net.minecraft.item.AliasedBlockItem;
 import net.minecraft.item.FoodComponent;
 import net.minecraft.item.Item;
@@ -129,6 +131,9 @@ public class Berries {
             //set berry types
             Item berryType = entry.getValue();
             bush.setBerryType(berryType);
+
+            //ensure bush is rendered with a cutout
+            BlockRenderLayerMap.INSTANCE.putBlock(bush.getBaseState().getBlock(), RenderLayer.getCutout());
 
             //register as compostable
             registerCompostableBerry(berryType);

--- a/src/main/java/io/ix0rai/bodaciousberries/registry/Berries.java
+++ b/src/main/java/io/ix0rai/bodaciousberries/registry/Berries.java
@@ -78,7 +78,7 @@ public class Berries {
      * map containing a berry bush, its associated berry, and unripe form if applicable
      * <br> should not be directly added to, use {@link #initialise(BerryBush, Item)}
      */
-    private static final Map<BerryBush, Item> BERRY_BUSHES = new HashMap<>();
+    public static final Map<BerryBush, Item> BERRY_BUSHES = new HashMap<>();
 
     /**
      * the chance that a berry will raise the level of compost in a composter
@@ -131,9 +131,6 @@ public class Berries {
             //set berry types
             Item berryType = entry.getValue();
             bush.setBerryType(berryType);
-
-            //ensure bush is rendered with a cutout
-            BlockRenderLayerMap.INSTANCE.putBlock(bush.getBaseState().getBlock(), RenderLayer.getCutout());
 
             //register as compostable
             registerCompostableBerry(berryType);

--- a/src/main/resources/fabric.mod.json
+++ b/src/main/resources/fabric.mod.json
@@ -20,6 +20,9 @@
   "entrypoints": {
     "main": [
       "io.ix0rai.bodaciousberries.Bodaciousberries"
+    ],
+    "client": [
+      "io.ix0rai.bodaciousberries.client.BodaciousberriesClient"
     ]
   },
   "mixins": [

